images/bugzilla-backporter/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM centos:7
```

Comment:
I also noticed that we use both 7 and 8. Is there a preferred version?

Reply:
Yeah, please use 8. It doesn't make a difference, because it will get replaced with https://github.com/openshift/release/pull/9880/files#diff-2aa47366feea3a8e22a64b1e867062adR62 which resolves to centos:8 but it makes it easier to understand